docs/achtergronddocumentatie/artefacten/0780.md

```diff
@@ -14,12 +14,13 @@ Deze oude issue kan vertaald worden naar de vraag of deze datums bij een klantco
 Nu zijn er in het klantcontavt een registratiedatum (vergelijkbaar met een creatiedatum) en een datum plaatsgevonden (vergelijkbaar met een indiendatum) opgenomen.
 
 Is het gewenst en nodig om de mutatiedatum op te nemen bij een klantcontact. 
-Het karakter van het klantdontact is dat het nietgewijzigd wordt. Er kan echter een reden zijn om één (of meerdere) correcties door te voeren. 
+Het karakter van het klantdontact is dat het niet gewijzigd wordt. Er kan echter een reden zijn om één (of meerdere) correcties door te voeren. 
 - Wat is het functionele belang van vastleggen van de mutatiedatum (of beter correctiedatum)?
 - Heb je het dan alleen over de laatste correctiedatum? 
 - Is de correctie zelf dan ook van belang?
 - Hier komen veel meer vragen rondom historie bij kijken
 
+Als een klantcontact vastgelegd is door een medewerker en (bv na het direct doorverbinden) daarna wordt aangepast door een 2e medewerker dan is dat weliswaar technsch een mutatie, maar functioneel betreft het één klantcontact. In dit geval is er geen noodzaak om een mutatiedatum vast te leggen.
 
 Ik snap dat je functionaliteit beschrijft vanuit de functionele wens, maar in hoeverre is bij de beschrijving van deze functies rekening gehouden met met technische implementeerbaarheid ? 
```

Comment:
Wat doet deze zin hier?

Reply:
Die stond er al in. @hdksi : Zal ik hem weghalen

Reply:
Besproken: zin kan weg.